chains/evm/contracts/libraries/Internal.sol

```diff
@@ -287,6 +287,8 @@ library Internal {
   // bytes4(keccak256("CCIP ChainFamilySelector APTOS"));
   bytes4 public constant CHAIN_FAMILY_SELECTOR_APTOS = 0xac77ffec;
 
+  bytes4 public constant CHAIN_FAMILY_SELECTOR_c4e05953 = 0xc4e05953;
```

Comment:
Please Better define what this chain family is supposed to be and what the pre-image is like above for Aptos